src/xdocs/config_javadoc.xml

```diff
@@ -1616,6 +1617,27 @@ public boolean isSomething()
 &lt;module name=&quot;SummaryJavadocCheck&quot;/&gt;
         </source>
         <p>
+          Example of <code>{@inheritDoc}</code> without summary.
+        </p>
+        <source>
+          <![CDATA[
+  public class Test extends Exception {
+    /**
+     * {@inheritDoc}  //Valid
```

Comment:
please place "//Valid" above javadoc comment. Examples should be ready to copy paste by user and works for config that is placed for it.
Please do the same for second javadoc too.